### DISCUSSION TITLE
Dynamic fields

### DISCUSSION THREAD

**Discussion by sbrow** [filamentphp/filament#348] [Help]
I don't know whether this should be a bug report or a feature request:

my code has a `FileUpload` component that accepts an XML file. I need to parse the uploaded file and create a list of `Select` fields based on RegExp matches from the file.

Whenever I try to do this, I get an error in the frontend `Uncaught ReferenceError: select is not defined`, meaning the Alpine.js component hasn't loaded yet?

Is there a way around this? Or are dynamic fields not currently supported?

I also tried creating a custom field type, but ran into the same issue.

**Comment by ryangjchandler** [comment]
@sbrow Dynamic fields in that nature aren't supported, no. The issue will be Livewire not respecting `@push` and `@pushonce` directives in subsequent requests.
